requirements/lint.in

```diff
@@ -14,4 +14,4 @@
 # The dependencies are sorted by alphabetical order.
 # Dependencies that do not come from pypi (eg. eggs from github) are listed at the end of the list.
 # -------------------------------------------------------------------------------------------------
-ruff==0.9.6
+ruff==0.12.1
```

Comment:
I just realized that I updated this, but not the .txt file 🤦🏼

Reply:
It would be nice to have a check for that.